tests/google-iot-cloud.log

```diff
@@ -0,0 +1,23 @@
+connected to mqtt.2030.ltsapis.goog\:8883
```

Comment:
Are we sure that the host name won't change? You could consider only matching the start of the message "connected to"

Reply:
I checked Google's documentation, and the value is constant. But I agree we can use regular expression to not specify the exact hostname.